lib/rubocop/cop/rails/time_zone_assignment.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of `Time.zone=` method.
```

Comment:
It may be useful for users to provide a summary of why good case is preferable in test :-)